packages/streamline/src/components/list-items/list-item-article/list-item-article.spec.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { fireEvent, waitFor } from '@testing-library/react-native';
+
+import { renderWithProvider } from '../../../testing/render-with-provider';
+
+import ListItemArticle from './list-item-article';
+
+describe('ListItemArticle', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should call onPress when pressed', async () => {
+    // Given
+    const onPress = jest.fn();
+    const { getByText } = renderWithProvider(
+      <ListItemArticle
+        imageSource={{
+          uri: 'https://reactnative.dev/img/tiny_logo.png',
+        }}
+        header="Header"
+        title="Title"
+        description="Description"
+        onPress={onPress}
+      />
+    );
+
+    // When
+    fireEvent.press(getByText('Header'));
+
+    // Then
+    await waitFor(() => expect(onPress).toHaveBeenCalledTimes(1));
```

Comment:
missing tests case:

- getByText('Title')
- getByText('Description')

And imageSource using accessibility text?